pandas/tests/test_groupby.py

```diff
@@ -4712,6 +4712,42 @@ def test_timegrouper_get_group(self):
                 result = grouped.get_group(dt)
                 assert_frame_equal(result, expected)
 
+    def test_timegrouper_apply_return_type_series(self):
+        # Using `apply` with the `TimeGrouper` should give the
+        # same return type as an `apply` with a `Grouper`.
+        df = pd.DataFrame({'date': ['10/10/2000', '11/10/2000'],
+                           'value': [10, 13]})
+        df_dt = df.copy()
+        df_dt['date'] = pd.to_datetime(df_dt['date'])
+
+        def sumfunc_series(x):
+            return pd.Series([x['value'].sum()], ('sum',))
+
+        grp_out = df.groupby(pd.Grouper(key='date')).apply(sumfunc_series)
```

Comment:
can you also tests that these produce the same output when doing a ``.resample(..).apply(..)`` (both functions)